Sources/BraintreeThreeDSecure/BTThreeDSecureAuthenticateJWT.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+class BTThreeDSecureAuthenticateJWT {
+
+    static func authenticate(
+        jwt cardinalJWT: String,
```

Comment:
Does this automatically translate to objective-c function name "authenticateJWT"?